internal/cmd/postgres/client.go

```diff
@@ -0,0 +1,39 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"entgo.io/ent/dialect/sql"
+	"github.com/openmeterio/openmeter/config"
+	"github.com/openmeterio/openmeter/internal/ent/db"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+type Clients struct {
+	Driver *sql.Driver
+	Client *db.Client
+}
+
+func GetClients(config config.PostgresConfig) (*Clients, error) {
```

Comment:
Q: do you think the subpackages are warranted? (i.e. `cmd/xy`), couldn't it all just be under `internal/cmd`? (do you think it reasonable to have multiple files per package?)

Reply:
I am fine merging those just I wanted to prevent this becoming a kitchen sink package.

Reply:
Splitting up later is always easier than merging things back, so I'm also on the side of keeping things together, until it doesn't make sense.

Reply:
I have dialed back the size of the refactor, given that what is the most problematic is the magic constants in the clickhouse options. In this [PR](https://github.com/openmeterio/openmeter/pull/1278) I have moved the options part to the config package (like we do for kafka) and then it's fine to initialize the clickhouse client in the main.